06-sdks/README.md

```diff
@@ -1,5 +1,5 @@
-CARTO SDKs
-==========
+CARTO ENGINE I: development kits
+=======================================
 
 # Agenda
 
```

Comment:
In the README.md file, there are two different documents in lines 14 and 15 (they point to the python intro slides and python jupyter notebook) that have the same name (Python SDK Introduction), maybe it should be better to change one of them to `Python SDK Introduction slides` or `Python SDK Introduction examples`?